action/action_test.go

```diff
@@ -0,0 +1,41 @@
+package action
+
+import (
+	"bytes"
+	"log"
+	"testing"
+	"time"
+
+	"github.com/damianoneill/nc-hammer/result"
+	"github.com/damianoneill/nc-hammer/suite"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Execute(t *testing.T) {
+	var buff bytes.Buffer
+	var want string
+	ts1, _ := suite.NewTestSuite("../suite/testdata/test-suite.yml")             // testsuite with netconf actions
```

Comment:
Never use identifiers with xx1, xx2, name the identifier so the user understands what it is for e.g. ts-valid or ts-invalid